tests/integration/create_product.t

```diff
@@ -23,15 +23,16 @@ my %product_feilds = (
 	origin => "france",
 	serving_size => "10g",
 	packaging_text => "no",
+	action => "process",
+	type => "add",
 	".submit" => "submit"
 );
 
-create_product($ua, \%product_feilds);
+create_user($admin_ua, {});
+create_product($admin_ua, \%product_fields);
 
-# edit preference accessible
-my $response = $ua->get("http://world.openfoodfacts.localhost/cgi/product.pl?type=edit&code=200000000099");
+my $response = $admin_ua->get("http://world.openfoodfacts.localhost/cgi/product.pl?type=edit&code=200000000099");
```

Comment:
It would be nice to also test an anomymous user can see it (with a new client)

Reply:
yes! added this check, it works